apps/kitchensink-react/e2e/document-list.spec.ts

```diff
@@ -11,30 +16,35 @@ test.describe('Document List', () => {
         name: `Test Author ${i}`,
         biography: `This is a test biography for author ${i}`,
       })),
+      {asDraft: false},
```

Comment:
This is an attempt to resolve a bit of flakiness, anecdotally I found that published documents were "visible" to queries a bit faster.